### PR TITLE
site: update dumi type

### PR DESCRIPTION
**Pull request by li-jia-nan**
[[中文版模板 / Chinese template](https://github.com/ant-design/ant-design/blob/master/.github/PULL_REQUEST_TEMPLATE/pr_cn.md?plain=1)]

### 🤔 This is a ...

- [ ] New feature
- [ ] Bug fix
- [x] Site / documentation update
- [ ] Demo update
- [ ] Component style update
- [ ] TypeScript definition update
- [ ] Bundle size optimization
- [ ] Performance optimization
- [ ] Enhancement feature
- [ ] Internationalization
- [ ] Refactoring
- [ ] Code style optimization
- [ ] Test Case
- [ ] Branch merge
- [ ] Workflow
- [ ] Other (about what?)

### 📝 Changelog

| Language   | Changelog |
| ---------- | --------- |
| 🇺🇸 English | 无需纳入ChangeLog |
| 🇨🇳 Chinese | 无需纳入ChangeLog |

### ☑️ Self-Check before Merge

⚠️ Please check all items below before requesting a reviewing. ⚠️

- [x] Doc is updated/provided or not needed
- [x] Demo is updated/provided or not needed
- [x] TypeScript definition is updated/provided or not needed
- [x] Changelog is provided or not needed

### CODE REVIEW

File: .dumi/theme/builtins/Previewer/index.tsx [hunk 1]
```diff
@@ -10,26 +10,19 @@ const Previewer = React.lazy(() => import('./Previewer'));
 const useStyle = createStyles(({ css }) => ({
   skeletonWrapper: css`
     width: 100% !important;
-    height: 250px;
+    height: 100%;
```
Comment (MadCcc) [inline .dumi/theme/builtins/Previewer/index.tsx:13]: 印象里这个不等价，Skeleton.Node 的 className 和 style 不是给到同一个元素的

Comment (MadCcc) [reply .dumi/theme/builtins/Previewer/index.tsx:13]: 这里的 250px 是必须的

Comment (li-jia-nan) [reply .dumi/theme/builtins/Previewer/index.tsx:13]: 我改回去

File: .dumi/theme/common/Link.tsx [hunk 2]
```diff
@@ -17,11 +17,11 @@ const Link = forwardRef<HTMLAnchorElement, LinkProps>((props, ref) => {
   const navigate = useNavigate();
   const { pathname } = useLocation();
 
-  const href = useMemo(() => {
+  const href = React.useMemo<string>(() => {
     if (typeof to === 'object') {
       return `${to.pathname || pathname}${to.search || ''}${to.hash || ''}`;
     }
-    return to;
+    return to || '';
```
Comment (MadCcc) [inline .dumi/theme/common/Link.tsx:24]: 感觉把 to 改成 required 更好

Comment (li-jia-nan) [reply .dumi/theme/common/Link.tsx]: done~

**Comment by li-jia-nan** [comment]
求 +1

**Review by afc163 — APPROVED** [review]

**Pull request merged**